weather_forecast_retrieval/data/hrrr/grib_file.py

```diff
@@ -80,47 +103,38 @@ def load(self, file, var_map):
             data = xr.open_dataset(
                 file,
                 engine='cfgrib',
-                backend_kwargs={'filter_by_keys': params}
+                backend_kwargs={
+                    'filter_by_keys': params,
+                    'indexpath': '',  # Don't create an .idx file when reading
+                }
             )
 
             if len(data) > 1:
                 raise Exception('More than one grib variable returned')
 
+            data = data.where(
```

Comment:
This reverses what was done prior? Reading the data, cropping to the area then assigning coordinates. Was this faster or just a more robust workflow?

Reply:
Correct. The test files are a bit too small to really be able to see a speed difference.

My motivation was to first subset the HRRR data to the area of interest and then do some index manipulation. Simply had that Xarray data cube in my head and thought it seems more robust to act on a smaller cube. :ice_cube:

Reply:
Makes sense.